x-pack/test/api_integration/apis/management/ingest_pipelines/lib/fixtures.ts

```diff
@@ -12,7 +12,7 @@ import {
   IngestPutPipelineRequest,
 } from '@elastic/elasticsearch/lib/api/types';
 
-interface Pipeline {
+export interface Pipeline {
```

Comment:
I can't see that this is used outside of this file?

Reply:
maybe it's no longer relevant, but it was used in another time when this was fixed :/

Reply:
```suggestion
interface Pipeline {
```